### PR TITLE
Added new methods of troubleshooting

### PR DESCRIPTION
**Pull request by Cybelyrium**
- Unable to mount CTRNAND or load the CTRNAND FIRM.
- Booted from CTRNAND via B9S (even if SD is inserted)
- "No accessible data in SD card" (BannerBomb3)
- Turning off parental controls
- No space for NAND backup
- More exception screens solutions (linking wiki)

### CODE REVIEW

File: _pages/en_US/troubleshooting.txt [hunk 1]
```diff
@@ -118,7 +118,7 @@ If neither of these solutions fixes this problem, then custom firmware may have
 <details>{{ compat | markdownify }}</details>
 
 {% capture compat %}
-<summary><u>DSiWare Management menu does not crash</u></summary>
+<summary><u>DSiWare Management menu does not crash or "No accessible software data."</u></summary>
```
Comment (Gruetzig) [inline _pages/en_US/troubleshooting.txt:121]: maybe add a verb like "or displays "No accesible software data." "

File: _pages/en_US/troubleshooting.txt [hunk 2]
```diff
@@ -525,6 +562,15 @@ ARM11 exception handlers are disabled, or custom firmware is not installed. Try
 {% endcapture %}
 <details>{{ compat | markdownify }}</details>
 
+{% capture compat %}
+<summary><u>My device gives me an exception screen when booting/loading an app</u></summary>
+
+Look for your exception screen in [this page](https://wiki.hacks.guide/wiki/3DS:Error_screens/Luma3DS_exception_screen).
+If you weren't able to find your error or you, for some reason, couldn't follow the given instructions, join blah [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp) for further assistance.
```
Comment (Gruetzig) [inline _pages/en_US/troubleshooting.txt:569]: You may want to remove the blah

File: _pages/en_US/troubleshooting.txt [hunk 3]
```diff
@@ -514,6 +545,12 @@ Your Luma3DS version is outdated. Download the latest release of [Luma3DS](https
 {% endcapture %}
 <details>{{ compat | markdownify }}</details>
 
+{% capture compat %}
+<summary><u>"Unable to mount CTRNAND or load the CTRNAND FIRM. Please use an external one."</u></summary>
+There is a number of reasons this could be happening. In any case, this error should be fixed by following the [CTRTransfer](ctrtransfer) guide.
```
Comment (Gruetzig) [inline _pages/en_US/troubleshooting.txt:550]: I would reword that to "There is a number of reasons for this to happen." or "There is a number of reasons as to why this could be happening."

File: _pages/en_US/troubleshooting.txt [hunk 4]
```diff
@@ -609,3 +655,18 @@ This can occur after a CTRTransfer or region change on Old 3DS / 2DS. You will n
 {% endcapture %}
 <details>{{ compat | markdownify }}</details>
 
+{% capture compat %}
+
+<summary><u>Turning off Parental Controls</u></summary>
```
Comment (Gruetzig) [inline _pages/en_US/troubleshooting.txt:660]: if you know the pin you can just disable it without the masterkey, this is nowhere mentioned and might be misleading

File: _pages/en_US/troubleshooting.txt [hunk 5]
```diff
@@ -453,6 +454,24 @@ You did not copy the `gm9` folder from the GodMode9 `.zip` to the root of your S
...
 
+{% capture compat %}
+<summary><u>"Backup failed" or "Error: Could not open directory" when attempting a NAND backup</u></summary>
+Make sure you have at least 1.3GB available in your SD card. If you don't have enough space, follow the steps:
+1. Power off your console
+1. Remove the SD card and insert it into your computer
+1. Copy the Nintendo 3DS folder from the root of your SD card to your computer
+1. Delete it from the SD card
+1. Load GodMode9 by holding START
+1. Perform the NAND Backup
```
Comment (Gruetzig) [inline _pages/en_US/troubleshooting.txt:465]: people might need more steps for getting back to the Backup Options

File: _pages/en_US/troubleshooting.txt [hunk 6]
```diff
@@ -453,6 +454,24 @@ You did not copy the `gm9` folder from the GodMode9 `.zip` to the root of your S
...
+1. Copy the files in gm9/out on your SD to a safe spot on your computer
+1. Delete the files from the SD card
```
Comment (Gruetzig) [inline _pages/en_US/troubleshooting.txt:467]: id prefer if only the nand backup bin and sha gets wiped

**Review by Gruetzig — CHANGES_REQUESTED** [review]

File: _pages/en_US/troubleshooting.txt [hunk 7]
```diff
@@ -459,14 +459,14 @@ You did not copy the `gm9` folder from the GodMode9 `.zip` to the root of your S
...
-1. Copy the Nintendo 3DS folder from the root of your SD card to your computer
+1. Copy the `Nintendo 3DS` folder from the root of your SD card to your computer
 1. Delete it from the SD card
 1. Load GodMode9 by holding START
-1. Perform the NAND Backup
-1. Copy the files in gm9/out on your SD to a safe spot on your computer
-1. Delete the files from the SD card
-1. Copy the Nintendo 3DS folder to your SD card root from your computer
-1. Delete the Nintendo 3DS folder from your computer
+1. Perform the [NAND Backup](godmode9-usage#restoring-a-nand-backup)
```
Comment (Gruetzig) [inline _pages/en_US/troubleshooting.txt:465]: restoring a nand backup isnt performing a nand backup

File: _pages/en_US/troubleshooting.txt [hunk 8]
```diff
@@ -525,6 +562,15 @@ ARM11 exception handlers are disabled, or custom firmware is not installed. Try
...
 
+{% capture compat %}
+<summary><u>My device gives me an exception screen when booting/loading an app</u></summary>
+
+Look for your exception screen in [this page](https://wiki.hacks.guide/wiki/3DS:Error_screens/Luma3DS_exception_screen).
+If you weren't able to find your error or you, for some reason, couldn't follow the given instructions, join [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp) for further assistance.
+{% endcapture %}
+<details>{{ compat | markdownify }}</details>
+
+
```
Comment (lifehackerhansol) [inline _pages/en_US/troubleshooting.txt:573]: unnecessary newline 2: electric boogaloo

File: _pages/en_US/troubleshooting.txt [hunk 9]
```diff
@@ -609,3 +655,19 @@ This can occur after a CTRTransfer or region change on Old 3DS / 2DS. You will n
 {% endcapture %}
 <details>{{ compat | markdownify }}</details>
 
+{% capture compat %}
+
```
Comment (lifehackerhansol) [inline _pages/en_US/troubleshooting.txt:659]: unnecessary newline 3: uh what's after electric boogaloo someone educate me

File: _pages/en_US/troubleshooting.txt (hunk 4 above)
Comment (lifehackerhansol) [reply _pages/en_US/troubleshooting.txt:660]: It kind of is mentioned, as it says do the mkey steps "if you do not know the PIN and it is stopping you from accessing the device's settings"

File: _pages/en_US/troubleshooting.txt [hunk 10]
```diff
@@ -118,7 +118,7 @@ If neither of these solutions fixes this problem, then custom firmware may have
 <details>{{ compat | markdownify }}</details>
 
 {% capture compat %}
-<summary><u>DSiWare Management menu does not crash</u></summary>
+<summary><u>DSiWare Management menu does not crash or displays "No accessible software data."</u></summary>
```
Comment (lifehackerhansol) [inline _pages/en_US/troubleshooting.txt:121]: I think we can just straight up remove the "does not crash" section. It will either crash, show a question mark, or show nothing.

Will leave it up to PR author to decide on that.

**Review by lifehackerhansol — CHANGES_REQUESTED** [review]

File: _pages/en_US/troubleshooting.txt (hunk 10 above)
Comment (lifehackerhansol) [reply _pages/en_US/troubleshooting.txt:121]: by section I mean just the title name, don't remove the whole troubleshooting lol

i.e. `s/does not crash//`

File: _pages/en_US/troubleshooting.txt (hunk 4 above)
Comment (Gruetzig) [reply _pages/en_US/troubleshooting.txt:660]: they fixed it in another line, when I submitted the review that text wasnt there. (this can be resolved)

File: _pages/en_US/troubleshooting.txt (hunk 10 above)
Comment (Gruetzig) [reply _pages/en_US/troubleshooting.txt:121]: I would suggest "and" instead of "or"

File: _pages/en_US/troubleshooting.txt [hunk 11]
```diff
@@ -453,6 +453,24 @@ You did not copy the `gm9` folder from the GodMode9 `.zip` to the root of your S
 {% endcapture %}
 <details>{{ compat | markdownify }}</details>
 
+{% capture compat %}
+<summary><u>"Backup failed" or "Error: Could not open directory" when attempting a NAND backup</u></summary>
+Make sure you have at least 1.3GB available in your SD card. If you don't have enough space, follow the steps:
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:458]: follow the steps -> follow these steps

File: _pages/en_US/troubleshooting.txt [hunk 12]
```diff
@@ -453,6 +453,24 @@ You did not copy the `gm9` folder from the GodMode9 `.zip` to the root of your S
...
+1. Power off your console
+1. Remove the SD card and insert it into your computer
+1. Copy the `Nintendo 3DS` folder from the root of your SD card to your computer
+1. Delete it from the SD card
+1. Load GodMode9 by holding START
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:463]: Use "power off, hold start, power on still holding start" as seen in rest of guide

File: _pages/en_US/troubleshooting.txt [hunk 13]
```diff
@@ -453,6 +453,24 @@ You did not copy the `gm9` folder from the GodMode9 `.zip` to the root of your S
...
+1. Perform the [NAND Backup](godmode9-usage#creating-a-nand-backup)
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:464]: the -> a, or just give people in-line instructions to create a nand backup to prevent getting redirected from page to page

File: _pages/en_US/troubleshooting.txt [hunk 14]
```diff
@@ -453,6 +453,24 @@ You did not copy the `gm9` folder from the GodMode9 `.zip` to the root of your S
 {% endcapture %}
 <details>{{ compat | markdownify }}</details>
 
+{% capture compat %}
+<summary><u>"Backup failed" or "Error: Could not open directory" when attempting a NAND backup</u></summary>
+Make sure you have at least 1.3GB available in your SD card. If you don't have enough space, follow the steps:
+1. Power off your console
+1. Remove the SD card and insert it into your computer
+1. Copy the `Nintendo 3DS` folder from the root of your SD card to your computer
+1. Delete it from the SD card
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:462]: clarity: it -> "the Nintendo 3DS folder"

File: _pages/en_US/troubleshooting.txt [hunk 15]
```diff
@@ -453,6 +453,24 @@ You did not copy the `gm9` folder from the GodMode9 `.zip` to the root of your S
...
+1. Load GodMode9 by holding START
+1. Perform the [NAND Backup](godmode9-usage#creating-a-nand-backup)
+1. Copy the files in `gm9/out` on your SD to a safe spot on your computer
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:465]: consistency: spot -> location

File: _pages/en_US/troubleshooting.txt [hunk 16]
```diff
@@ -453,6 +453,24 @@ You did not copy the `gm9` folder from the GodMode9 `.zip` to the root of your S
...
+1. Delete the `<date>_<serialnumber>_sysnand_##.bin` and `<date>_<serialnumber>_sysnand_##.bin.sha` files from the SD card, keeping essential.exefs
+1. Copy the `Nintendo 3DS` folder to your SD card root from your computer
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:467]: Copy the `Nintendo 3DS` folder from your computer to the root of your SD card

File: _pages/en_US/troubleshooting.txt [hunk 17]
```diff
@@ -453,6 +453,24 @@ You did not copy the `gm9` folder from the GodMode9 `.zip` to the root of your S
...
+<summary><u>"Backup failed" or "Error: Could not open directory" when attempting a NAND backup</u></summary>
+Make sure you have at least 1.3GB available in your SD card. If you don't have enough space, follow the steps:
+1. Power off your console
+1. Remove the SD card and insert it into your computer
+1. Copy the `Nintendo 3DS` folder from the root of your SD card to your computer
+1. Delete it from the SD card
+1. Load GodMode9 by holding START
+1. Perform the [NAND Backup](godmode9-usage#creating-a-nand-backup)
+1. Copy the files in `gm9/out` on your SD to a safe spot on your computer
+1. Delete the `<date>_<serialnumber>_sysnand_##.bin` and `<date>_<serialnumber>_sysnand_##.bin.sha` files from the SD card, keeping essential.exefs
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:466]: Specify that these files are in `/gm9/out/`

File: _pages/en_US/troubleshooting.txt [hunk 18]
```diff
@@ -505,6 +523,19 @@ The steps below can be attempted in any order, but are listed from least to most
 {% endcapture %}
 <details>{{ compat | markdownify }}</details>
 
+{% capture compat %}
+<summary><u>My device does not show any of my apps on the HOME Menu</u></summary>
+
+This could be caused by various reasons, but mostly because your SD card is not being read by the system
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:529]: Missing period from end of sentence

File: _pages/en_US/troubleshooting.txt [hunk 19]
```diff
@@ -505,6 +523,19 @@ The steps below can be attempted in any order, but are listed from least to most
...
+You can check if your SD is not being read by holding SELECT on boot and checking the yellow text on the bottom screen; If it says "Booted from CTRNAND via B9S", then your system is booting from the internal memory and not from the SD card.
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:530]: use lowercase after semicolon, but a period would work fine here too

Comment (lilyuwuu) [reply _pages/en_US/troubleshooting.txt:530]: also, this could indicate that boot.firm is missing from the sd card for whatever reason

Comment (lilyuwuu) [reply _pages/en_US/troubleshooting.txt:530]: stylistically speaking, bullet points do not usually use a period at the end

File: _pages/en_US/troubleshooting.txt [hunk 20]
```diff
@@ -505,6 +523,19 @@ The steps below can be attempted in any order, but are listed from least to most
...
+Attempt the steps below in any order:
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:531]: "in any order" leads to "what order do I follow this in"; try to convey that these are the possible troubleshooting steps from easiest to hardest

File: _pages/en_US/troubleshooting.txt [hunk 21]
```diff
@@ -505,6 +523,19 @@ The steps below can be attempted in any order, but are listed from least to most
...
+1. Power off your device, remove the SD card, re-insert it, then power on your device.
+1. Power off your device, remove the SD card, insert it on your PC and format your SD card according to your system: [Windows](formatting-sd-(windows)), [macOS](formatting-sd-(mac)), [Linux](formatting-sd-(linux)).
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:533]: format -> reformat (maybe)
Maybe warn that this will wipe SD card data

File: _pages/en_US/troubleshooting.txt [hunk 22]
```diff
@@ -505,6 +523,19 @@ The steps below can be attempted in any order, but are listed from least to most
...
+1. Test your SD card by following the guide according to your system: [Windows](h2testw-(windows), [Linux](f3-(linux)), [macOS](f3xswift-(mac)). If your SD card is marked as faulty, then you will have to replace your SD card.
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:534]: add "for errors" to "Test your SD card"

Comment (lilyuwuu) [reply _pages/en_US/troubleshooting.txt:534]: system -> computer's operating system

File: _pages/en_US/troubleshooting.txt (hunk 21 above)
Comment (lilyuwuu) [reply _pages/en_US/troubleshooting.txt:533]: system -> computer's operating system

File: _pages/en_US/troubleshooting.txt [hunk 23]
```diff
@@ -505,6 +523,19 @@ The steps below can be attempted in any order, but are listed from least to most
...
+1. Your SD card slot is broken. Join [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp) for further assistance.
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:535]: SD card slot *may be* broken

File: _pages/en_US/troubleshooting.txt [hunk 24]
```diff
@@ -514,6 +545,12 @@ Your Luma3DS version is outdated. Download the latest release of [Luma3DS](https
 {% endcapture %}
 <details>{{ compat | markdownify }}</details>
 
+{% capture compat %}
+<summary><u>"Unable to mount CTRNAND or load the CTRNAND FIRM. Please use an external one."</u></summary>
+There are a number of reasons as to why this could be happening. In any case, this error should be fixed by following the [CTRTransfer](ctrtransfer) guide.
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:550]: should be fixed -> can usually be fixed? slightly different nuance, personal stylistic choice

File: _pages/en_US/troubleshooting.txt [hunk 25]
```diff
@@ -525,6 +562,14 @@ ARM11 exception handlers are disabled, or custom firmware is not installed. Try
 {% endcapture %}
 <details>{{ compat | markdownify }}</details>
 
+{% capture compat %}
+<summary><u>My device gives me an exception screen when booting/loading an app</u></summary>
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:566]: "My device gives me an" is not necessary, but might be useful/kept for clarity
app -> application

File: _pages/en_US/troubleshooting.txt [hunk 26]
```diff
@@ -525,6 +562,14 @@ ARM11 exception handlers are disabled, or custom firmware is not installed. Try
...
+
+Look for your exception screen in [this page](https://wiki.hacks.guide/wiki/3DS:Error_screens/Luma3DS_exception_screen).
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:568]: in -> on (maybe)

File: _pages/en_US/troubleshooting.txt [hunk 27]
```diff
@@ -525,6 +562,14 @@ ARM11 exception handlers are disabled, or custom firmware is not installed. Try
...
+If you weren't able to find your error or you, for some reason, couldn't follow the given instructions, join [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp) for further assistance.
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:569]: "you, for some reason," slightly antagonistic; use 'instructions didn't work"

File: _pages/en_US/troubleshooting.txt [hunk 28]
```diff
@@ -609,3 +654,18 @@ This can occur after a CTRTransfer or region change on Old 3DS / 2DS. You will n
 {% endcapture %}
 <details>{{ compat | markdownify }}</details>
 
+{% capture compat %}
+<summary><u>Turning off Parental Controls</u></summary>
+
+You can disable the Parental Controls feature by going to System Settings -> Parental Controls and inserting the PIN, then pressing "Clear Settings" and "Delete" to remove it.
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:660]: clear settings, *then* delete (comma required?)

File: _pages/en_US/troubleshooting.txt [hunk 29]
```diff
@@ -609,3 +654,18 @@ This can occur after a CTRTransfer or region change on Old 3DS / 2DS. You will n
...
+However, if you do not know the PIN and it is stopping you from accessing the device's settings, you will need to disable it. In order to do this, you need to obtain your device's master key (mkey):
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:661]: and it is stopping you from -> and therefore cannot

File: _pages/en_US/troubleshooting.txt [hunk 30]
```diff
@@ -609,3 +654,18 @@ This can occur after a CTRTransfer or region change on Old 3DS / 2DS. You will n
...
+1. Access [this website](https://mkey.eiphax.tech/)
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:662]: access -> go to

File: _pages/en_US/troubleshooting.txt [hunk 31]
```diff
@@ -609,3 +654,18 @@ This can occur after a CTRTransfer or region change on Old 3DS / 2DS. You will n
...
+1. Fill the following spaces with the information:
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:663]: spaces -> boxes

File: _pages/en_US/troubleshooting.txt [hunk 32]
```diff
@@ -609,3 +654,18 @@ This can occur after a CTRTransfer or region change on Old 3DS / 2DS. You will n
...
++ Device Type: The console you are currently using
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:664]: will be 3ds in all cases, being vague will lead to "i have a 2ds and it didn't listed do i select dsi"

**Review by lilyuwuu — CHANGES_REQUESTED** [review]
mostly just grammatical/style suggestions

**Review by lifehackerhansol — APPROVED** [review]
